src/pages/SupplierRequisitionPage.js

```diff
@@ -37,10 +38,15 @@ const MODAL_KEYS = {
 export class SupplierRequisitionPage extends React.Component {
   constructor(props) {
     super(props);
+
+    const { requisition } = props;
```

Comment:
@joshxg 
introduced unused variable ? this should be enforced in pre commit eslint error right ?

Reply:
It's used on the next line

Reply:
oops ma bad